packages/next/src/server/config.ts

```diff
@@ -739,7 +721,18 @@ function assignDefaults(
     result.experimental = {}
   }
   result.experimental.optimizePackageImports = [
-    ...new Set([...userProvidedOptimizePackageImports, 'lucide-react']),
+    ...new Set([
+      ...userProvidedOptimizePackageImports,
+      'lucide-react',
+      '@headlessui/react',
+      '@fortawesome/fontawesome-svg-core',
```

Comment:
@shuding I'm running into an issue with this one, specifically with the CSS import in my `app/layout.ts`:

```ts
// app/layout.ts
import { config } from "@fortawesome/fontawesome-svg-core";
import '@fortawesome/fontawesome-svg-core/styles.css'

config.autoAddCss = false;
```

Importing the fontawesome styles.css is working fine on `v13.4.19`, but breaks on this commit (`v13.4.20-canary.13`). Let me know if you need a repro.

Reply:
We've fixed this in a follow up PR (https://github.com/vercel/next.js/pull/55040#pullrequestreview-1612790074), thanks!